src/main.rs

```diff
@@ -113,6 +121,7 @@ impl ProvingService for CloudProver {
             };
         };
 
+        #[allow(dead_code)]
```

Comment:
This directive prevents lint (clippy) warnings about un-read values.

Reply:
Similar to my suggestion for `SindriTaskStatusResponse`, I'd propose we just remove the following two fields instead of making an allowance:
```
            circuit_id: String,
            circuit_name: String,
```

Reply:
Fields removed and struct renamed in [cb58816](https://github.com/Sindri-Labs/sindri-scroll-sdk/pull/17/commits/cb588166d2aa383c68e2130b54424f63cfeaf9e5).

Reply:
One more line to remove here to ensure we aren't overriding the lint warning:
```suggestion
```